cypress/e2e/git/branch_menu.feature

```diff
@@ -17,3 +17,24 @@ Feature: Test git branch menu
     When I click on "[data-cy=\"git-current-branch\"]"
     Then I expect "[data-cy=\"git-menu-branch-local-master\"]" exists
     And I expect "[data-cy=\"git-menu-branch-local-master\"] [data-cy=\"git-menu-current-branch\"]" exists
+
+  Scenario: Expect to don't have any branch on "no_master" filter
```

Comment:
`to don't` => `Expect to have no branch with ...`

Reply:
done